src/applications/debt-letters/components/ExpandableAlert.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import recordEvent from 'platform/monitoring/record-event';
+
+const ExpandableAlert = ({ label, content, status, iconType, ...rest }) => {
+  const [open, setOpen] = useState(false);
+
+  const handleOnclick = e => {
+    if (!content) return;
+    if (e) e.target.classList.toggle('active');
```

Comment:
This should also toggle an [`aria-expanded`](https://www.w3.org/WAI/GL/wiki/Using_aria-expanded_to_indicate_the_state_of_a_collapsible_element) and include an [`aria-controls`](https://www.w3.org/WAI/GL/wiki/Using_the_WAI-ARIA_aria-expanded_state_to_mark_expandable_and_collapsible_regions) to indicate what the button is controlling

Reply:
Makes total sense, thanks for the great feedback as always!

Reply:
@Mottie feedback is implemented and double checked with Axe